doc-surrealdb_versioned_docs/version-latest/surrealql/functions/database/index.mdx

```diff
@@ -15,96 +15,119 @@ SurrealDB comes with a large number of in-built functions for checking, manipula
   <thead>
     <tr>
       <th scope="col">Module</th>
-      <th scope="col">Example / Notes</th>
+      <th scope="col">Description</th>
+      <th scope="col">Example</th>
     </tr>
   </thead>
   <tbody>
     <tr>
-      <td scope="row" data-label="Module"><a href="/docs/surrealdb/surrealql/functions/database/array"><code>Array Functions</code></a></td>
+      <td scope="row" data-label="Module"><a href="/docs/surrealdb/surrealql/functions/database/array"><code>Array functions</code></a></td>
```

Comment:
suggestion: we can get rid of the `functions` suffix for each module as we are in the `Database Functions` page already

Reply:
I personally agree with this.

Reply:
My pushback here would be that just having Array, for example vs Array Functions would only make sense if the Modules column title is changed to Functions.

Reply:
Better change the column name to `Functions` then.

Reply:
I changed the column name to "Function Module" which seems like the closest description of the column to me. Let me know if you want it changed.